### DISCUSSION TITLE
Neptune X Robin Nano V1.3

### DISCUSSION THREAD

**Discussion by nvd0620** [NARUTOfzr/ZNP-Robin-Nano-V1.2#11] [Q&A]
What changes would it take to get this running on a Neptune X equipped with a Robin Nano I got from my Neptune 2s?

**Comment by NARUTOfzr** [comment]
https://github.com/NARUTOfzr/Neptune_X-VGT6-VET6/tree/main/Configuration%20files/Neptune%20X
Loading the "elegoo.txt" file should do the job
